data/monsters/tob.js

```diff
@@ -31,32 +60,7 @@ const tob = {
 		const loot = [];
 		for (let i = 0; i < quantity; i++) {
 			if (this.roll(650)) loot.push(this.drops.pet);
-			if (this.roll(40)) {
-				const number = parseInt((Math.random() * 20) + 1);
-				switch (true) {
-					case number === 1:
-						loot.push(this.drops.scytheOfVitur);
-						break;
-					case number <= 3:
-						loot.push(this.drops.sanguinestiStaff);
-						break;
-					case number <= 5:
-						loot.push(this.drops.ghraziRapier);
-						break;
-					case number <= 7:
-						loot.push(this.drops.justiciarFaceguard);
-						break;
-					case number <= 9:
-						loot.push(this.drops.justiciarLegguards);
-						break;
-					case number <= 11:
-						loot.push(this.drops.justiciarChestguard);
-						break;
-					default:
-						loot.push(this.drops.avernicDefenderHilt);
-						break;
-				}
-			}
+			if (this.roll(9.1)) loot.push(this.determineItem().emoji);
```

Comment:
The `.1` of this is not effecting the chance of the roll. Also, the `this.determineItem().emoji` gives an error on pretty much every command usage as `this.determineItem()` doesnt return anything, and it tries to read `undefined.emoji`.

`.1` is used elsewhere in the changes too. You can not use `roll` at all and just make a custom RNG method for CoX based on however it works.

Reply:
well thats what i get for not trying to fix my local build to even run, that if in determineItem was leftover that i didn't delete when i initially built it, sorry about that.

as far as i can tell the `.1` will actually do something in the roll method since its multiplied before the floor